tgui/packages/tgui/styles/interfaces/DecalPainter.scss

```diff
@@ -0,0 +1,8 @@
+.DecalPainter__CellButton {
+  margin: 2px !important;
+  padding: 0.5em !important;
+}
+
+.DecalPainter__CellIcon {
+  display: block !important;
```

Comment:
It looks like you don't need this style since you can simply use a div for the icon, and that has display block by default.